index.js

```diff
@@ -183,6 +185,52 @@ class GithubScm extends Scm {
         };
     }
 
+    /**
+     * Promise to wait a certain number of seconds
+     *
+     * Might make this centralized for other tests to leverage
+     *
+     * @method promiseToWait
+     * @param  {Number}      timeToWait  Number of seconds to wait before continuing the chain
+     * @return {Promise}
+     */
+    promiseToWait(timeToWait) {
+        return new Promise((resolve) => {
+            setTimeout(() => resolve(), timeToWait * 1000);
+        });
+    }
+
+    /**
+     * Get PR mergeability
+     * @async  getPrMergeable
+     * @param  {Object}   config
+     * @param  {String}   config.scmUri     The scmUri to get PR info of
+     * @param  {String}   config.token      The token used to authenticate to the SCM
+     * @param  {Integer}  config.prNum      The PR number used to fetch the PR
+     * @param  {Integer}  count             The polling count
+     * @return {Promise}
+     */
+    async getPrMergeable({ scmUri, token, prNum }, count) {
+        if (count >= POLLING_MAX_ATTEMPT) {
+            winston.warn('Computing mergerbility did not finish');
+
+            return { success: false };
```

Comment:
Do we need these return value? these are not used from caller.
If these are for testing, we should add some comment.

Reply:
I added comment to JSDoc.